genienlp/tasks/dialogue_dataset.py

```diff
@@ -47,3 +47,7 @@ def return_splits(cls, path='.data', train='train', validation='valid', test='te
         return Split(train=train_data, eval=validation_data, test=test_data), Split(
             train=train_path, eval=validation_path, test=test_path
         )
+
+
+class ErrorClassificationDataset(E2EDialogueDataset):
```

Comment:
If this class is specific to dialogue datasets (which I think it is, given that it inherits from `E2EDialogueDataset`, it should have "Dialogue" in its name.